export/usb.py

```diff
@@ -3,13 +3,18 @@
 import re
 from subprocess import CalledProcessError, check_call
 import logging
-
 from pyudev import Context, Monitor
 
-video_pattern = re.compile('missed-moment.*merged.*mp4')
-MEDIA_DIR = '/missed_moment_media'
-USB_MOUNT_DIR = '/missed_moment_usb'
-USB_DIR = USB_MOUNT_DIR + '/missed_moment'
+# import constants
+import sys
+sys.path.append('..')
+import constants
+
+
+VIDEO_PATTERN = re.compile(constants.VIDEO_PATTERN)
+MEDIA_DIR = constants.MEDIA_DIR
+USB_MOUNT_DIR = constants.USB_MOUNT_DIR
+USB_DIR = constants.USB_DIR
```

Comment:
I wouldn't declare new variables for these. Just use `constants.foo` where you need to use it. (`VIDEO_PATTERN` is the exception. That is correct to have here)

Reply:
Ok I made this change globally as well